### PR TITLE
Add chat trigger

### PR DESCRIPTION
**Pull request by rom1504**

### CODE REVIEW

File: lib/generate-packets.js [hunk 1]
```diff
@@ -37,4 +37,10 @@ async function triggerTabComplete (server, bot) {
   await p
 }
 
+async function triggerChat (server, bot) {
+  const p = once(bot, 'message')
```
Comment (u9g) [inline lib/generate-packets.js:41]: isnt there a chat event too? maybe we can promise.all with that one too

[ref](https://minecraft-data.prismarine.js.org/?v=1.16.1&d=protocol#toClient_chat)

Comment (rom1504) [reply lib/generate-packets.js:41]: one is enough no ?

**Comment by rom1504** [comment]
looking at https://github.com/PrismarineJS/prismarine-packet-dumper/pull/42#issuecomment-822064667  it was already covered

File: lib/generate-packets.js (hunk 1 above)
Comment (u9g) [reply lib/generate-packets.js:41]: ah nvm, maybe i see now that mineflayer's message event is triggered by the nmp chat event

**Review by u9g — APPROVED** [review]
lgtm

**Pull request merged**